### PR TITLE
[DEV-1622] Add more details to the epics returned from /filter/jira

### PR DESCRIPTION
**Pull request by vmarkovtsev**

### CODE REVIEW

File: server/athenian/api/openapi/openapi.yaml [hunk 1]
```diff
@@ -5524,82 +5524,126 @@ components:
...
-          description: When the issue was marked as completed. This timestamp can
+          description: When the issue finished. This timestamp can
             be missing and is always greater than or equal to `work_began`.
           format: date-time
           type: string
+        type:
+          description: Name of the issue's type.
+          type: string
+        prs:
+          description: Number of mapped pull requests.
```
Comment (warenlg) [inline server/athenian/api/openapi/openapi.yaml:5631]: This is really cool, it's a important asset of the feature. Could we add the overall aggregation to the epic layer? eg epic DEV-245 -> 45 PRs overall

Btw, I assume we talk about all PRs (open and closed), right?

File: server/athenian/api/openapi/openapi.yaml [hunk 2]
```diff
@@ -5657,6 +5701,27 @@ components:
...
+        name: Done
+        stage: finished
+      properties:
+        name:
+          description: Exact status name.
+          type: string
+        stage:
+          description: One of the three status categories.
+          enum:
+          - backlog
```
Comment (warenlg) [inline server/athenian/api/openapi/openapi.yaml:5717]: The 3 Jira status categories are officially called: To Do, In Progress, Done
see https://athenianco.atlassian.net/secure/admin/ViewStatuses.jspa

File: server/athenian/api/openapi/openapi.yaml [hunk 3]
```diff
@@ -5524,82 +5524,126 @@ components:
...
+          type: integer
       required:
       - id
+      - title
+      - created
+      - updated
+      - reporter
+      - comments
       - status
       - type
```
Comment (warenlg) [inline server/athenian/api/openapi/openapi.yaml:5644]: The ticket type object should contain the ticket icon url

**Review by warenlg — COMMENTED** [review]
Nothing critical, some comments and minor adjustments here and there.

File: server/athenian/api/openapi/openapi.yaml [hunk 4]
```diff
@@ -5524,82 +5524,126 @@ components:
...
+          type: string
+        comments:
+          description: Number of comments in the issue including sub-tasks.
+          type: integer
+        priority:
+          $ref: '#/components/schemas/JIRAPriority'
+        status:
+          $ref: '#/components/schemas/JIRAStatus'
         work_began:
           description: When the issue entered the "In Progress" stage. This timestamp
```
[the remark this answers is not in the document]
Comment (vmarkovtsev) [reply server/athenian/api/openapi/openapi.yaml:5614]: Yep, that's how it already works, I should change the description.

File: server/athenian/api/openapi/openapi.yaml (hunk 1 above)
Comment (vmarkovtsev) [reply server/athenian/api/openapi/openapi.yaml:5631]: Yes, everything that's mapped.

**Review by warenlg — APPROVED** [review]

**Comment by vmarkovtsev** [comment]
Good! I am able to make this backward-compatible.

**Pull request merged**